analytics_data_api/v0/views/courses.py

```diff
@@ -688,14 +688,15 @@ class VideosListView(BaseCourseView):
 
     **Response Values**
 
-        Returns a collection of video views and metadata for each video. Each collection contains:
+        Returns a collection of video views and metadata for each video. 
```

Comment:
There is a linting error here and looks to be caused by a trailing white space. at the end of this line.